src/Components/Welcome/Welcome.tsx

```diff
@@ -17,7 +17,7 @@ const Welcome = () => {
       </div>
       <div className='link-container'>
         <Link className="play-game-link" to={{pathname: `/game/${(JSON.parse(localStorage.getItem("selectedTheme") || '{}')).value || "random"}`}}>Play Game!</Link>
```

Comment:
This Link tag should be self-closing. Meaning if we want the "Play Game!" text we'd have to wrap it in something else like a <p> tag.

Reply:
Does this apply to the Link in `EndGame.tsx`?